src/logging.rs

```diff
@@ -17,20 +17,24 @@ thread_local! {
 }
 
 pub fn init() {
+    let start_time = clock::now();
     env_logger::Builder::new()
-        .format(|fmt, record| {
-            CONTEXT.with(|ctxt| {
+        .format(move |fmt, record| {
+            CONTEXT.with(move |ctxt| {
                 let level = match record.level() {
                     Level::Trace => "TRCE",
                     Level::Debug => "DBUG",
                     Level::Info => "INFO",
                     Level::Warn => "WARN",
                     Level::Error => "ERR!",
                 };
+                let uptime = clock::now() - start_time;
```

Comment:
How concerned are we about the performance impact of adding timestamps? 

I believe that (unlike `SystemTime::now`) whether or not `clock::now()` will require a syscall every time may be platform-dependant, but there's still some added overhead.

If this is a concern, we could instead include timestamps in the messages of certain log records where knowing the time of the event matters (such as the identity daemon's log messages)...

Reply:
I'd like to have the uptime counter if possible--it helps to understand relative timing.

How can we determine whether `clock::now` invokes a syscall? We have a pretty specific target platform...

Reply:
By default, I believe it depends on the behaviour of `Instant::now`. IIRC, someone (possibly @carllerche?) told me at one point that in some cloud provider's environment, `Instant::now` always invoked a syscall but in others it didn't. Sorry I don't recall all the details around this OTTOMH.

Reply:
I would not worry about it until it is a problem.